src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java

```diff
@@ -2091,6 +2091,12 @@ static Response setDataverseLogo(String dataverseAlias, String pathToImageFile,
                 .multiPart("file", new File(pathToImageFile))
                 .put("/api/dataverses/" + dataverseAlias + "/logo");
     }
+    
+    static Response setCollectionAttribute(String dataverseAlias, String attribute, String value, String apiToken) {
```

Comment:
I'm excited to have this method and functionality. In future test cases, we could use it in DataversesIT to test changing the name, description, and affiliation of collections.

We COULD do that now, in this PR, if we feel like it. It's new functionality that should probably be exercised.

Reply:
As I mentioned in the description, I'm copy-and-pasting/stealing this API from Oliver's pr that's still in draft (#9462). This by itself is going to introduce at least a minor merge conflict with his branch. @poikilotherm are you ok with me expanding on it in this branch, like adding an integration test? - or do you want me to leave it to you?

Reply:
(I added a simple test that uses the API to change the alias then makes sure the change has taken effect)

Reply:
Please go ahead! Im happy to resolver any merge conflicts that might appear.